src/io/srt_io.ml

```diff
@@ -141,12 +141,14 @@ object(self)
 
   method virtual id : string
 
+  method self_sync = true
+
   val mutable clock = None
   method private get_clock =
     match clock with
       | Some c -> c
       | None ->
-         let c = new Clock.self_sync self#id in
+         let c = new Clock.clock self#id in
```

Comment:
I would have expected to have
```suggestion
         let c = new Clock.clock "srt" in
```
as everywhere else. Maybe the creation function should take both a string an (optional labeled) id?